example/server/integrations.py

```diff
@@ -140,6 +141,7 @@ def check_kyc(transaction: Transaction, post_data=None) -> Optional[Dict]:
                     "continue on this page."
                 ),
                 "icon_label": _("Stellar Development Foundation"),
+                "icon_path": "images/lobstr.png",
```

Comment:
i don't think we should use a third party companies logo in our example

Reply:
Oh yea that was a test. We don't have a PNG file to use, so we'll just use the Polaris default.